input/docs/cs/affine_picture_transformation.md

```diff
@@ -0,0 +1,34 @@
+﻿title: Afinní transformace obrázku pomoci rotace
+--- 
+Transformační matice T je matice o velikosti 3x3, která se používá k provedení operace posunu, rotaci a odražení objektů v afinní transformaci 2D obrázků vůči původním souřadnicím osy.
+
+ ![](../media/2018-11-20-14-33-14.jpg)
+
+Rotace obrázků se provádí proti směru hodinových ručiček. Matice T popisuje otáčení bodu ve původních souřadnicích.
+
+		T = ( cos(φ) -sin(φ)  
+		      sin(φ)  cos(φ) )
+Pro rotaci po směru hodinových ručiček je nutné se počítat se zápornou hodnotou uhlu φ.
+		
+		T = ( cos(φ) sin(φ)  
+		     -sin(φ) cos(φ) )
+Funkce affine2d(T) přirázuje matici vlastnosti platnou afinní transformací matice T. Výsledná afinní transformace obrázku se provádí pomoci funkci imwarp(), kde se obrázek mění podle definovaného formátu rotaci.
+
+		(X' Y') = (X Y) * (cos(φ) -sin(φ)   
```

Comment:
Co je X, X' ? K čemu se ta rovnice váže? ... Proč to tu vůbec je?  (tohle souvisí s tím komentářem o tom jak se provádí rotace s pomocí transformační matice.)